commit db94403c0167bb38f580541df0faae7daf380d24
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Jan 24 15:19:39 2018 +0000

    lan78xx: Avoid spurious kevent 4 "error"
    
    lan78xx_defer_event generates an error message whenever the work item
    is already scheduled. lan78xx_open defers three events -
    EVENT_STAT_UPDATE, EVENT_DEV_OPEN and EVENT_LINK_RESET. Being aware
    of the likelihood (or certainty) of an error message, the DEV_OPEN
    event is added to the set of pending events directly, relying on
    the subsequent deferral of the EVENT_LINK_RESET call to schedule the
    work.  Take the same precaution with EVENT_STAT_UPDATE to avoid a
    totally unnecessary error message.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/net/usb/lan78xx.c b/drivers/net/usb/lan78xx.c
index 69bb657e0f44..92e3bdb9435b 100644
--- a/drivers/net/usb/lan78xx.c
+++ b/drivers/net/usb/lan78xx.c
@@ -2528,7 +2528,7 @@ static void lan78xx_init_stats(struct lan78xx_net *dev)
 	dev->stats.rollover_max.eee_tx_lpi_transitions = 0xFFFFFFFF;
 	dev->stats.rollover_max.eee_tx_lpi_time = 0xFFFFFFFF;
 
-	lan78xx_defer_kevent(dev, EVENT_STAT_UPDATE);
+	set_bit(EVENT_STAT_UPDATE, &dev->flags);
 }
 
 static int lan78xx_open(struct net_device *net)
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1035_db94403c0167bb38f580541df0faae7daf380d24.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1035_db94403c0167bb38f580541df0faae7daf380d24.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
